rainbowsaddle/__init__.py

```diff
@@ -103,7 +103,8 @@ def wait_pid(self, pid):
         """
         try:
             os.waitpid(pid, 0)
-        except OSError, err:
+        except OSError:
```

Comment:
@flupke Have changed it.

Don't know if it's appropriate to pull request to the master branch.